src/platform/jwt.ts

```diff
@@ -61,6 +61,7 @@ export interface User extends sbvrUtils.User {
 
 	twoFactorRequired?: boolean;
 	authTime?: number;
+	password?: string;
```

Comment:
I don't think this should be here, this is about a req.user / jwt user which should definitely not include the password, not a pinejs user resource

Reply:
@Page- thanks for the heads up. It looked weird to me as well.
Saw the `jwt_secret` in there and got confused.
I will create a new interface for the DbUser. Maybe actually this should be JwtUser and the DbUser should be just User... will check how it looks, but let me know what you think.